app/controllers/users/sessions_controller.rb

```diff
@@ -31,10 +31,9 @@ def active
     end
 
     def timeout
-      if sign_out
-        analytics.track_event(Analytics::SESSION_TIMED_OUT)
-        flash[:timeout] = t('session_timedout')
-      end
+      analytics.track_event(Analytics::SESSION_TIMED_OUT)
+      sign_out
```

Comment:
seemed like the previous version depended on the return value of `sign_out` but this one assumes success. The examples I see in the devise repo suggest the return value is irrelevant, so seems ok to me.

https://github.com/plataformatec/devise/search?p=2&q=sign_out&utf8=%E2%9C%93